Source/WebCore/Modules/mediastream/MediaStream.cpp

```diff
@@ -369,14 +374,14 @@ Document* MediaStream::document() const
     return downcast<Document>(scriptExecutionContext());
 }
 
-void MediaStream::stop()
+void MediaStream::inactivate()
```

Comment:
considering this is always called by `stop()` and this new `inactivate` is called by `void LibWebRTCMediaEndpoint::stop()` shouldn't we leave it as is ?

Reply:
stop is ActiveDOMObject::stop. I think it is clearer this way.